acceptance/workspace/jobs/create/output.txt

```diff
@@ -0,0 +1,5 @@
+
+>>> $CLI jobs create --json {"name":"abc"}
```

Comment:
We could autogenerate these acceptance tests for most commands in the CLI. Ensures incrementally nothing breaks because of SDK bump etc. Or at least we have a record.